src/ui/templates/results/verticalresultscount.hbs

```diff
@@ -1,19 +1,16 @@
 {{#unless isHidden}}
-  <div class="yxt-VerticalResultsCount"
-    aria-label="{{translate phrase='[[start]] through [[end]] of [[resultsCount]]' start=pageStart end=pageEnd resultsCount=total }}">
-    <div aria-hidden="true">
+  <div class="yxt-VerticalResultsCount">
       {{translate
         phrase=
-'<span class="yxt-VerticalResultsCount-start">[[start]]</span>
-<span class="yxt-VerticalResultsCount-dash">-</span>
-<span class="yxt-VerticalResultsCount-end">[[end]]</span>
-<span class="yxt-VerticalResultsCount-of">of</span>
-<span class="yxt-VerticalResultsCount-total">[[resultsCount]]</span>'
+    '<span class="yxt-VerticalResultsCount-start">[[start]]</span>
```

Comment:
Is it necessary to add a tab to these rows? Since that changed the msgid, that requires us to update all of the .po files to match it. I believe that's what caused the tests to fail